home-assistant-plugin/custom_components/climate/xknx.py

```diff
@@ -35,6 +35,8 @@
 CONF_OVERRIDE_SUPPORTED_OPERATION_MODES = 'override_supported_operation_modes'
 CONF_ON_OFF_ADDRESS = 'on_off_address'
 CONF_ON_OFF_STATE_ADDRESS = 'on_off_state_address'
+CONF_OVERRIDE_MIN_TEMP = 'override_min_temp'
+CONF_OVERRIDE_MAX_TEMP = 'override_max_temp'
```

Comment:
Why not just `CONF_MIN_TEMP` ? If it is specified as parameter, you override it. So it is redundant.

Reply:
Thanks will change it!

I did it this way initially because it might confuse people who are using the setpoint shift calculations as this will override any of these changes. I might change this a bit.